src/components/invoice/invoice-edit/EditInvoice.tsx

```diff
@@ -87,6 +92,21 @@ export class EditInvoice extends Component<EditInvoiceProps, EditInvoiceState> {
     return this.isQuotation ? 'quotation' : 'invoice';
   }
 
+  getConsultantName(invoice: InvoiceModel) {
```

Comment:
Ter info:
Zet ook een return type bij functies
Hier is het nogal evident dat het string is natuurlijk, maar altijd handig als je direct aan de signatuur kan zien wat het ding returnt